fastai/data/load.py

```diff
@@ -18,7 +17,10 @@ def _wif(worker_id):
     ds.wif()
 
 class _FakeLoader:
-    _IterableDataset_len_called,_auto_collation,collate_fn,drop_last = None,False,noops,False
+    def fn_noops(self, x=None, *args, **kwargs):
+        "Do nothing (method)"
```

Comment:
Suggest removing the docstring and putting function def all on one line, since it's not really needed externally

Reply:
![image](https://user-images.githubusercontent.com/16190118/106350776-7c3f0c80-6312-11eb-94f7-79d9f9e25ac6.png)